samples/sensor/vl53l0x/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_I2C=y
+CONFIG_GPIO=y
+CONFIG_SENSOR=y
+CONFIG_VL53L0X=y
+CONFIG_VL53L0X_PROXIMITY_THRESHOLD=100
+
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Is NEWLIB_LIBC really needed?

Reply:
This one is needed for abs function

Reply:
abs can be a one-liner macro.

Reply:
yes, I used it before in the first versions of this PR...
For an other topic, I discovered NEWLIB_LIBC and see that it include abs. So my conclusion was to use the official way instead of redefining existing functions.

If this is the last blocking point, I can reimplement this function. Else, I prefer to keep NEWLIB_LIBC.

Reply:
Nope, just a thought, not blocking, can be left for later, if someone ever has a real issue with it.